src/components/DashboardLayout/LeftNav.tsx

```diff
@@ -25,8 +27,9 @@ export default function LeftNav({ className }: { className?: string }) {
   }, 1000);
 
   return (
```

Comment:
Fix scrolling behavior. The current one is very chaotic
<img width="182" alt="image" src="https://user-images.githubusercontent.com/44907675/218779496-19c37aa0-ae31-4888-8efc-1fd1732f5e95.png">